ebu_tt_live/bindings/validation/timing.py

```diff
@@ -265,8 +309,38 @@ def _semantic_manage_timeline(self, dataset, element_content):
         # Register on timeline
         doc.add_to_timeline(self)
 
+
     # This section covers the copying operations of timed containers.
 
+    def _semantic_validate_active_areas(self, dataset):
+        # Get the document instance
+        doc = dataset['document']
+        if self.computed_begin_time is not None and self.computed_end_time is not None:
+            affected_elements = doc.lookup_range_on_timeline(self.computed_begin_time, self.computed_end_time)
+            if len(affected_elements) > 1:
+                for elem1, elem2 in itertools.combinations(affected_elements, 2):
+                  if elem1 != elem2:
+                    if elem1.region is not None and elem2.region is not None: #checking if the elements have regions
+                        # Getting coordinates from the attribute eg ["14% 16%"]
+                        elem1_region = dataset["elements_by_id"][elem1.region]
+                        elem2_region = dataset["elements_by_id"][elem2.region]
+                        origins_1 = elem1_region.origin.split(" ")
+                        origins_2 = elem2_region.origin.split(" ")
+                        l1 =  [float(origin.strip('%')) for origin in origins_1] 
+                        l2 =  [float(origin.strip('%')) for origin in origins_2] 
+                        extents_1 = elem1_region.extent.split(" ")
+                        extents_2 = elem1_region.extent.split(" ")
+                        r1 = [float(extent.strip('%')) for extent in extents_1] 
+                        r2 = [float(extent.strip('%')) for extent in extents_2] 
+                        # Checking for overlapping rectangles
+                        if l1[0] < r2[0] and r1[0] > l2[0] and l1[1] > r2[1] and r1[1] < l2[1]:
+                            raise OverlappingActiveElementsError(self)
+    
+    def _semantic_validate_is_timed_leaf(self, dataset):
+        if (len(self.span)) > 0:
+            self.begin = None
```

Comment:
I added this here because once a new d_p_type is made then it is validated, so if I removed the timings in the ebutt3_ebuttd.py before it is validated, then when the validation for overlapping regions happen there wont be a p begin or end time if they have spans.